packages/build/tests/plugins/api/snapshots/tests.js.md

```diff
@@ -273,20 +273,18 @@ Generated by [AVA](https://ava.li).
     ␊
     ␊
     ┌─────────────────────────────┐␊
-    │     Netlify Build Error     │␊
+    │        Build failed         │␊
     └─────────────────────────────┘␊
     ␊
-    Plugin "netlify-plugin-test" failing with errors␊
+    > Error message␊
```

Comment:
We need to remove the `api` value from passing into plugins e.g. `onInit({ api }) => {}`

We can re-enable once the api client has more methods